.chloggen/add-actions-to-headerssetter.yaml

```
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: headerssetter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Extend the headers setter extension with header modification actions.
+
+# One or more tracking issues related to the change
+issues: [16581, 7596]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  Update the existing headers setter configuration with `action: insert`.
```

Is this correct? I'd say to update, you'd use the action `update`, correct?
Do we want to mention the other actions too?
```suggestion
  Update the existing headers setter configuration:
        - `insert`: Inserts the new header if it does not exist.
        - `update`: Updates the header value if it exists.
        - `upsert`: Inserts a header if it does not exist and updates the header
          if it exists.
        - `delete`: Deletes the header.
```

---

Right, it is `update`, I'll push a fix-up.